Source/Core/DolphinQt2/Config/Graphics/SoftwareRendererWidget.cpp

```diff
@@ -161,7 +161,12 @@ void SoftwareRendererWidget::AddDescriptions()
       QT_TR_NOOP("Dump decoded game textures to User/Dump/Textures/<game_id>/.\n\nIf unsure, leave "
                  "this unchecked.");
 
+  static const char* TR_DUMP_OBJECTS_DESCRIPTION =
```

Comment:
`const char[]`

Reply:
Did you mean just drop the `static` part? VS didn't like the brackets. I also assume I should change the other `static const char* TR_` to `const char* TR_`?

Reply:
oh, I meant this should be a `static const char TR_DUMP_OBJECTS_DESCRIPTION[]`. (The brackets come after the name for arrays)